### PR TITLE
[FEATURE] Add frontend linting

### PR DESCRIPTION
**Pull request by lukaszuznanski**
This PR is the resurrection of #74 (after the switch from master to main).

### CODE REVIEW

File: Resources/Public/JavaScript/Main.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+document.addEventListener("DOMContentLoaded", function () {
+    var t = document.querySelectorAll(".tx-tea table"),
+        i = t.length;
+    while (--i >= 0) makeSortable(t[i]);
+});
+
+function makeSortable(table) {
```
Comment (oliverklee) [inline Resources/Public/JavaScript/Main.js:7]: Should we namespace the functions to avoid name collisions, e.g., like this?
https://github.com/oliverklee/ext-seminars/blob/master/Resources/Public/JavaScript/FrontEnd/FrontEnd.js#L22

File: Resources/Public/JavaScript/Main.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+document.addEventListener("DOMContentLoaded", function () {
+    var t = document.querySelectorAll(".tx-tea table"),
+        i = t.length;
+    while (--i >= 0) makeSortable(t[i]);
```
Comment (oliverklee) [inline Resources/Public/JavaScript/Main.js:4]: Please let's have the `{` and `}` here. (Shouldn't eslint have complained about this?)

File: Resources/Public/JavaScript/Main.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        (function (i) {
+            var dir = 1;
+            th[i].addEventListener("click", function () {
+                sortTable(table, i, (dir = 1 - dir));
+            });
+        })(i);
+}
+
+function sortTable(table, col, reverse) {
+    var tb = table.tBodies[0], //
```
Comment (oliverklee) [inline Resources/Public/JavaScript/Main.js:23]: Why the `//` here?

File: Resources/Private/Templates/Tea/Index.html [hunk 4]
```diff
@@ -8,6 +8,12 @@ <h2>
         </h2>
 
         <table class="table">
+            <thead>
+                <tr>
+                    <th>a</th>
+                    <th>b</th>
```
Comment (oliverklee) [inline Resources/Private/Templates/Tea/Index.html:14]: Please let's have meaningful headings here. I propose the localized labels "UID" and "Title".

File: .gitattributes [hunk 5]
```diff
@@ -7,4 +7,10 @@
 /.php_cs.php export-ignore
 /Tests/ export-ignore
 /codeception.yml export-ignore
+/phpcs.xml.dist export-ignore
 /phpcs.xml export-ignore
+/Resources/Private/.eslintignore export-ignore
+/Resources/Private/.eslintrc export-ignore
```
Comment (oliverklee) [inline .gitattributes:13]: This needs to be `/Resources/Private/.eslintrc.json` (with the added `.json`).

File: Resources/Private/.htaccess [hunk 6]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (oliverklee) [inline Resources/Private/.htaccess:1]: Why this change?

**Review by oliverklee — CHANGES_REQUESTED** [review]
I have carried over my comments from #74.

File: Resources/Private/Templates/Tea/Index.html (hunk 4 above)
Comment (lukaszuznanski) [reply Resources/Private/Templates/Tea/Index.html:14]: Looks like I submitted "test" content :)

File: Resources/Private/.htaccess (hunk 6 above)
Comment (lukaszuznanski) [reply Resources/Private/.htaccess:1]: miss click :)

**Review by oliverklee — CHANGES_REQUESTED** [review]
The JS linter has something to complain about. :-)

I also have updated the German translation with an additional commit.

File: .gitattributes [hunk 7]
```diff
@@ -7,4 +7,10 @@
 /.php_cs.php export-ignore
 /Tests/ export-ignore
 /codeception.yml export-ignore
+/phpcs.xml.dist export-ignore
```
Comment (oliverklee) [inline .gitattributes:10]: We don't have this file anymore.

**Comment by lukaszuznanski** [comment]
> The JS linter has something to complain about. :-)
> 
> I also have updated the German translation with an additional commit.

done :)

**Review by oliverklee — APPROVED** [review]
Looks good!
